### PR TITLE
Add TSC voting record for 2024-03-14 and update docs accordingly

### PR DESCRIPTION
**Pull request by gctucker**
Add an entry to the TSC voting records for the 2024-03-14 monthly and add Shuah to the list of TSC members as this is effective on Monday 2024-03-18.

### CODE REVIEW

File: kernelci.org/content/en/docs/org/tsc/votes.md [hunk 1]
```diff
@@ -35,6 +37,23 @@ outcome and a more detailed section listing each individual vote.
...
+**Motion: Add TSC Member: Shuah Khan**
+* Result: approved
+* Effective date: 2024-03-18
+* Voting method: TSC monthly
+* Voting members: arisut, broonie, gtucker, nuclearcat, pawiecz, spbnick
+* Vote count: 6/6/8 in favour
+
+**Motion: Elect TSC Chair**
+* Candidates: Shuah Khan, Nikolai Kondrashov
+* Result: approved
```
Comment (padovan) [inline kernelci.org/content/en/docs/org/tsc/votes.md:51]: `Result: Nikolai elected` instead ?

Comment (gctucker) [reply kernelci.org/content/en/docs/org/tsc/votes.md:51]: The motion to elect a new char was approved so this is correct afaict.  There's a line with the vote counts for each candidate, it's a bit of a special case as it's not a binary yes / no vote.

**Review by padovan — APPROVED** [review]

File: kernelci.org/content/en/docs/org/tsc/votes.md (hunk 1 above)
Comment (gctucker) [reply kernelci.org/content/en/docs/org/tsc/votes.md:51]: I could add a line with "Elected: Nikolai Kondrashov" if it's not obvious enough from the vote counts.

**Comment by gctucker** [comment]
I've added the "Elected: Nikolai Kondrashov" line below the "Candidates:" one for extra clarity.

**Comment by gctucker** [comment]
Thanks for the review.

@nuclearcat Please merge ahead of Monday's production update if this looks OK on staging etc.

File: kernelci.org/content/en/docs/org/tsc/_index.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ respective email address and IRC nicknames:
 * [Denys Fedoryshchenko](mailto:<denys.f@collabora.com>) - `nuclearcat`
 * [Jeny Sadadia](mailto:jeny.sadadia@collabora.com) - `jenysadadia`
 * [Paweł Wieczorek](mailto:pawiecz@collabora.com) - `pawiecz`
+* [Shuah Khan](mailto:skhan@linuxfoundation.org) - `shuah`
```
Comment (gctucker) [inline kernelci.org/content/en/docs/org/tsc/_index.md:31]: Verified on staging: https://static.staging.kernelci.org/docs/org/tsc/#members

File: kernelci.org/content/en/docs/org/tsc/votes.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ outcome and a more detailed section listing each individual vote.
 
 | Date                      | Motion                                                          | Status    |
 |---------------------------|-----------------------------------------------------------------|-----------|
+| [2024-03-14](#2024-03-14) | Add TSC member: Shuah Khan                                      | Approved  |
+| [2024-03-14](#2024-03-14) | Elect new TSC chair                                             | Approved  |
```
Comment (gctucker) [inline kernelci.org/content/en/docs/org/tsc/votes.md:15]: Verified on staging: https://static.staging.kernelci.org/docs/org/tsc/votes/#summary

File: kernelci.org/content/en/docs/org/tsc/votes.md [hunk 4]
```diff
@@ -35,6 +37,24 @@ outcome and a more detailed section listing each individual vote.
...
+
+**Motion: Elect TSC Chair**
+* Candidates: Shuah Khan, Nikolai Kondrashov
+* Elected: Nikolai Kondrashov
+* Result: approved
+* Effective date: 2024-04-01
+* Voting method: TSC monthly
+* Voting members: arisut, broonie, gtucker, nuclearcat, pawiecz, spbnick
+* Vote count: 4/6/8 for Nikolai, 1/6/8 for Shuah, 1/6/8 abstained
+
```
Comment (gctucker) [inline kernelci.org/content/en/docs/org/tsc/votes.md:57]: Verified on staging: https://static.staging.kernelci.org/docs/org/tsc/votes/#2024-03-14

**Review by gctucker — COMMENTED** [review]
Verified on staging.

**Comment by nuclearcat** [comment]
Merging

**Comment by gctucker** [comment]
FYI I pinged Shuah on Friday to double check the details (email address, IRC nickname etc.).  If there's any typo or anything needs fixing we should still be able to do that early next week I guess.  Maybe the steps to add a TSC member should include an email confirmation with a pending PR, especially for people who aren't regular GitHub users.

**Comment by padovan** [comment]
I reached out to her on Thursday already to get her details.

**Pull request merged**